### DISCUSSION TITLE
Use heap storage with ceph external to OKD

### DISCUSSION THREAD

**Discussion by dborgi** [rook/rook#12819] [Q&A]
My OKD cluster is version 4.13

I installed externally to OKD ceph storage in version 18.2

![image](https://github.com/rook/rook/assets/48066051/e5d2d3ea-ffa2-485c-b846-8ff98cea0b3a)

I need to make a storageclass for the okd logging, I was trying with room but I can't get it to connect to my external ceph, what guide can I use ? thank you

**Comment by parth-gr** [comment]
Are you facing problem to login to ceph dasboard,
Or connecting to rook cluster?

**Reply by dborgi** [reply]
Hello, I have no problems connecting to the ceph dashboard, if I make a storageclass from my okd to the external ceph

**Comment by travisn** [comment]
@dborgi You are following the [guide to configure the external cluster](https://rook.github.io/docs/rook/v1.12/CRDs/Cluster/external-cluster/)?

**Reply by dborgi** [reply]
Yes

**Reply by travisn** [reply]
The [troubleshooting guide](https://rook.io/docs/rook/latest/Troubleshooting/ceph-common-issues/) may have some ideas, but they are not specific to the external clusters. Do you see any errors in the rook operator log? Is the CSI driver running? Did you create the configmap with the mon endpoints? Any other details you can provide?
